manifests/a/ArobasMusic/GuitarPro/7/Soundbanks/1.1.123/ArobasMusic.GuitarPro.7.installer.yaml

```
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ArobasMusic.GuitarPro.7.Soundbanks
+PackageVersion: 1.1.123
+Platform:
+  - Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+UpgradeBehavior: install
+Installers:
+  - Architecture: x86
```

```suggestion
  - Architecture: x64
```
Since `x86` validations run on 32-bit OSes, I'm guessing this might be a 64-bit application.

---

[The publisher says](https://support.guitar-pro.com/hc/en-us/articles/360001599877-GP7-6-Download-Guitar-Pro-7-6-Tablature-Editor-Software) _Architecture 32 or 64 bits_. The main software (that uses the same installer) `ArobasMusic.GuitarPro.7` has been accepted with `Architecture: x86`. So I think it must be `neutral` or `x86`.